src/components/navi-button/navi-button.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import { NaviButtonProps } from './types';
+import clsx from 'clsx';
+import { Label, LabelSize } from '../typography';
+import { BaseButton } from '../../base/base-button/base-button';
+import { IconSize } from '../../elements/icons/types';
+
+export const NaviButton: FC<NaviButtonProps> = ({ onClick, className, Icon, label }) => {
```

Comment:
Also add the name prop optional, for example `name = 'navi-button'` and pass it down to the BaseButton